lib/registration.pm

```diff
@@ -262,8 +262,15 @@ sub registration_bootloader_params {
 }
 
 sub yast_scc_registration {
-
-    type_string "yast2 scc; echo yast-scc-done-\$?- > /dev/$serialdev\n";
+    my ($type_slow) = @_;
+    my $yast_scc_cmd = "yast2 scc; echo yast-scc-done-\$?- > /dev/$serialdev\n";
+    #Due to ninja keys on PPC, may be required to type command slowlier, see poo#19230
+    if ($type_slow) {
```

Comment:
do you have an idea why we *should* type slower in this specific case and not in general? There is also the variable VNC_TYPING_LIMIT to limit overall typing speed. And we have set this for ppc64le already so maybe we really need to lower the limit. Unless it was about "QA-Power-…" behaving weird, which is another progress ticket